docs/reference/modules/login.md

```diff
@@ -24,7 +24,7 @@ Defaults to `true`. If set to `false`, the normal login form at `/login` is comp
 
 ### `passwordReset`
 
-Defaults to `false`. If both `localLogin` and `passwordReset` are set to `true`, a password reset link is added to the login form. Clicking on this link brings up a form for the user to enter the email address associated with their account. The address is compared to the list of existing user emails and a reset link is sent if that email is found. Note, the site email settings must be correctly configured prior to using the password reset. See the [email guide](/guide/sending-email.html) for help in configuring the module. It is also recommended that the `baseUrl` be set in the `app.js` file. The user will not be notified if the email check fails or if the email fails to send due to improper configuration. However, debug information will output in the terminal.
+Defaults to `false`. If both `localLogin` and `passwordReset` are set to `true`, a password reset link is added to the login form. Clicking on this link brings up a form for the user to enter the email address associated with their account. The address is compared to the list of existing user emails and a reset link is sent if that email is found. Note, the site email settings must be correctly configured prior to using the password reset. See the [email guide](/guide/sending-email.html) for help in configuring the module. It is also recommended that the `baseUrl` be set in the `app.js` file or through the `APOS_BASE_URL` environment variable. The user will not be notified if the email check fails or if the email fails to send due to improper configuration. However, debug information will output in the terminal.
```

Comment:
app.js isn't new here but again probably not best practice unless it's conditional on something. `data/local.js` is environment specific when using stagecoach. In a more modern deployment system that's not a great solution, so we go with the environment variable.